utils/run_test.sh

```diff
@@ -68,6 +68,15 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES=${DAOS_BASE}/test_results/*
+    for f in $FILES
+    do
+        SUITE=`grep "testsuite name=" $f | grep -Po "name=\"\K.*(?=\" time=)"`
```

Comment:
(lint) Use $(..) instead of legacy `..`. [SC2006]
(lint) Double quote to prevent globbing and word splitting. [SC2086]